dts/bindings/usb-c/usb-c-connector.yaml

```diff
@@ -137,31 +168,107 @@ properties:
           Minimum power, in microwatts, needed by the sink. A Capability
           Mismatch is sent to the Source if the power can't be met.
 
-  # EXAMPLE:
-  #
-  # USB-C connector attached to a STM32 UCPD typec port controller, which has
-  # power delivery support and enables SINK.
-  #
-  # #include <dt-bindings/usb-c/pd.h>
-  #
-  # vbus1: vbus {
-  #     compatible = "zephyr,usb-c-vbus-adc";
-  #     io-channels = <&adc2 8>;
-  #     output-ohms = <49900>;
-  #     full-ohms = <(330000 + 49900)>;
-  # };
-  #
-  # ports {
-  #     #address-cells = <1>;
-  #     #size-cells = <0>;
-  #     port1: usb-c-port@1 {
-  #         compatible = "usb-c-connector";
-  #         reg = <1>;
-  #         tcpc = <&ucpd1>;
-  #         vbus = <&vbus1>;
-  #         power-role = "SINK";
-  #         sink-pdos = <PDO_FIXED(5000, 2000, PDO_FIXED_USB_COMM)
-  #                      PDO_VAR(5000, 12000, 2000)>;
-  #         op-sink-microwatt = <10000000>;
-  #      };
-  # };
+    epr-supported-as-snk:
```

Comment:
How did you decide which of these fields to put in the port node vs the VIF node?

Reply:
I was relying on information required to build VIF xml, the properties per port are being added in USB-C connector node, the properties common to all ports are added to VIF node. Following is the VIF XML schema: https://compliance.usb.org/cv/VendorInfoFile/Schemas/Current/VendorInfoFile.xsd , there Component refers to a port. There could be multiple components in a VIF file belonging to a product. Thank you